From 327e0a7bd9864043b855d93fc0b6be099d319dd0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Thu, 20 Mar 2025 19:17:17 +0100
Subject: [PATCH] Add more supported shell versions

There was no change in the code, so there should be no issues to keep
version 47 supported. Also, this extension works fine with GNOME 49 without
any changes.
---
 weatheroclock@CleoMenezesJr.github.io/metadata.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/weatheroclock@CleoMenezesJr.github.io/metadata.json b/weatheroclock@CleoMenezesJr.github.io/metadata.json
index c5bce22..2851815 100644
--- a/weatheroclock@CleoMenezesJr.github.io/metadata.json
+++ b/weatheroclock@CleoMenezesJr.github.io/metadata.json
@@ -2,7 +2,7 @@
   "uuid": "weatheroclock@CleoMenezesJr.github.io",
   "name": "Weather O'Clock",
   "description": "Display the current weather inside the pill next to the clock. GNOME Weather is required for this extension to work.",
-  "shell-version": ["48"],
+  "shell-version": ["47", "48", "49"],
   "url": "https://github.com/CleoMenezesJr/weather-oclock",
   "original-author": "JasonLG1979@github.io",
   "donations": {
